tests/LinearCodeAddressGenerator_test.cdc

```diff
@@ -0,0 +1,112 @@
+import Test
+import "LinearCodeAddressGenerator"
+
+access(all)
+fun setup() {
+    var err: Test.Error? = Test.deployContract(
+        name: "LinearCodeAddressGenerator",
+        path: "../contracts/LinearCodeAddressGenerator.cdc",
+        arguments: [],
+    )
+    Test.expect(err, Test.beNil())
+}
+
+access(all)
+fun generateAddresses(
+    count: UInt64,
+    chainCodeWord: UInt64
+): [Address] {
+    let addresses: [Address] = []
+    for index in InclusiveRange<UInt64>(1, count) {
+        let address = LinearCodeAddressGenerator.address(
+            at: index,
+            chainCodeWord: chainCodeWord
+        )
+        addresses.append(address)
+    }
+
+    return addresses
+}
+
+access(all)
+fun checkAddresses(
+    count: UInt64,
+    chainCodeWord: UInt64,
+    expected: [Address]
+) {
+     let actual = generateAddresses(
+        count: 10,
+        chainCodeWord: chainCodeWord
+    )
+
+    Test.assertEqual(expected, actual)
+
+    for address in actual {
+        Test.assert(
+            LinearCodeAddressGenerator.isValidAddress(
```

Comment:
An `isValidAddress` implementation that always returns true would pass this test. 
It would be good to add another assertion that
```
LinearCodeAddressGenerator.isValidAddress(
   address,
   chainCodeWord: otherChainCodeWord
)
```
returns `false`. 
We could set `otherChainCodeWord` to one chain different than `chainCodeWord`.

Reply:
Good idea! Added a test for the other chain code words in e5496c6b2dcae9769130a115b7be448878235668